tests/cql/CqlAggregateFunctionsTest.xml

```diff
@@ -68,13 +72,23 @@
 			<expression>AnyTrue({})</expression>
 			<output>false</output>
 		</test>
+		<test name="AnyTrueIsFalseWhenNull">
+			<expression>AnyTrue(null)</expression>
+			<output>false</output>
+		</test>
 	</group>
 	<group name="Avg">
 		<test name="AvgTest1">
 			<expression>Avg({ 1.0, 2.0, 3.0, 6.0 })</expression>
 			<output>3.0</output>
 		</test>
 	</group>
+	<group name="Product">
+		<test name="Product_Long">
+			<expression>Product({5L, 4L, 5L})</expression>
+			<output>100L</output>
+		</test>
```

Comment:
should we add other product tests (e.g. for other overloads, null input, etc) or should that come in another PR? (same comment for the other operators)

Reply:
+1 here if we're going to add this test group we should have a few more edge cases covered